node-registrar/client/node.go

```diff
@@ -59,6 +59,8 @@ type nodeCfg struct {
 	twinID        uint64
 	status        string
 	healthy       bool
+	online        *bool
+	lastSeen      *int64
```

Comment:
why using pointer not the actual value?

Reply:
The inline filter can be trure => online, false => offline, null => all nodes 

Last seen can be empty, to also include all results